packages/trader/src/AppV2/Components/OnboardingGuide/onboarding-video.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Skeleton } from '@deriv-com/quill-ui';
+import { getUrlBase } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const OnboardingVideo = () => {
+    const [is_loading, setIsLoading] = React.useState(true);
+
+    // memoize file paths for videos and open the modal only after we get them
+    const getVideoSource = React.useCallback(
+        (extension: string) => getUrlBase(`/public/videos/user-onboarding-guide-trade-page.${extension}`),
+        []
+    );
+    const mp4_src = React.useMemo(() => getVideoSource('mp4'), [getVideoSource]);
+
+    return (
+        <React.Fragment>
+            {is_loading && <Skeleton.Square height={218.5} />}
+            <video
+                autoPlay
+                data-testid='dt_onboarding_guide_video'
+                muted
+                loop
+                onLoadedData={() => setIsLoading(false)}
+                playsInline
+                preload='auto'
+                height={218.5}
```

Comment:
For accumulator stats video we not giving absolute height and width of the video and handling by aspect ratio . 
I think it would be same for this video as well.

Reply:
Will add, thank you